### PR TITLE
Update replicate-data-using-dm.md (#328)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #328 to release-1.0

---

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/304
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Have version specific changes 
- [x] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/replicate-data-using-dm.md [hunk 1]
```diff
@@ -48,9 +48,21 @@ After the DM cluster is deployed using DM-Ansible, the configuration information
     source-id = "mysql-replica-01"
     dm-worker = "172.16.10.72:8262"
 
+<<<<<<< HEAD
```
Comment (yikeke) [inline en/replicate-data-using-dm.md:51]: Please resolve conflicts according to the zh version. @ireneontheway

**Review by yikeke — CHANGES_REQUESTED** [review]

File: en/replicate-data-using-dm.md (hunk 1 above)
Comment (ireneontheway) [reply en/replicate-data-using-dm.md]: Fixed in https://github.com/pingcap/docs-dm/pull/332/commits/e8de4204e482f74bce0685ceefd6ce6a5b6f34f8

Comment (csuzhangxc) [reply en/replicate-data-using-dm.md]: I think it should match #308 but no diff now.

Comment (TomShawn) [reply en/replicate-data-using-dm.md]: @ireneontheway https://github.com/pingcap/docs-dm/pull/308 FIY.

File: en/replicate-data-using-dm.md [hunk 2]
```diff
@@ -48,21 +48,9 @@ After the DM cluster is deployed using DM-Ansible, the configuration information
     source-id = "mysql-replica-01"
     dm-worker = "172.16.10.72:8262"
```
Comment (ireneontheway) [inline en/replicate-data-using-dm.md]:
```suggestion
    # This indicates that whether DM-worker uses Global Transaction Identifier (GTID) to pull binlog. Before you use this configuration item, make sure that the GTID mode is enabled in the upstream MySQL.
    enable-gtid: false
```

File: en/replicate-data-using-dm.md (hunk 1 above)
Comment (ireneontheway) [reply en/replicate-data-using-dm.md]: Fixed in https://github.com/pingcap/docs-dm/pull/332/commits/ccc99ad75ca71a82a21575c448350498e28eac19

File: en/replicate-data-using-dm.md [hunk 3]
```diff
@@ -43,6 +43,9 @@ After the DM cluster is deployed using DM-Ansible, the configuration information
 
     ```toml
     # Master configuration.
+    
+    # This indicates that whether DM-worker uses Global Transaction Identifier (GTID) to pull binlog. Before you use this configuration item, make sure that the GTID mode is enabled in the upstream MySQL.
+    enable-gtid: false
```
Comment (csuzhangxc) [inline en/replicate-data-using-dm.md:48]:
```suggestion
    enable-gtid = false
```

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**